bundle/tasks/install-conductr-haproxy.yml

```diff
@@ -3,16 +3,20 @@
   find: paths="/usr/share/conductr/extra" patterns="conductr-haproxy-v*.zip"
   register: conductr_haproxy_bundle
 
-- name: Load Bundle - ConductR HAProxy
-  command: "conduct load {{ conductr_haproxy_bundle.files[0].path }}"
-
 - shell: "hostname -i"
   register: host_ip
 
-- name: Get HAProxy Nodes
-  conductr_members_get: role="haproxy" conductr_control_host="{{ host_ip.stdout }}"
-  register: haproxy_nodes
+- name: Load Bundle - ConductR HAProxy
+  command: "conduct load --ip {{ host_ip.stdout }} {{ conductr_haproxy_bundle.files[0].path }}"
```

Comment:
Would you consider modifying the `conductr_members_get` function?

* At the moment the function hits the `/members` endpoint, so I think it can be modified to look for `/agents` instead.
* The function should be renamed `conductr_agents_get` instead after the change.

Once this change is done, the function will obtain a list of agent address given a particular role.

Using `/agents` will work since we have core and agent sitting on the same physical machine with haproxy enabled to all machines. However, if we were to change this setup and enable roles for the agents, simply counting the number of agents in the `/agents` endpoint will yield incorrect information.

Alternatively, if you feel using `/agents` is ok for now, perhaps we should consider deleting `conductr_members_get` function.